applications/GeoMechanicsApplication/tests/test_normal_load_on_hexa_element.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import test_helper
+
+
+class KratosGeoMechanicsNormalLoadHexaTests(KratosUnittest.TestCase):
+    """
+    This class contains tests for applying normal loads on 3D elements.
+    """
+
+    def test_hexa_8n_normal_load(self):
+        test_name = 'hexa_8n_normal_load'
+        file_path = test_helper.get_file_path(test_name)
+        test_helper.run_kratos(file_path)
+
+        all_integration_points = [0, 1, 2, 3, 4, 5, 6, 7]
```

Comment:
You could do it this way, but you could achieve the same effect when you don't provide any integration point list (i.e. leaving the corresponding function parameter at its default value).